### PR TITLE
Add ibm.storage_virtualize, which ibm.spectrum_virtualize was renamed to

### LINKED ISSUE #306: Rename included ibm.spectrum_virtualize to ibm.storage_virtualize
Relates to https://github.com/ansible-collections/ansible-inclusion/discussions/65

The renaming process is described in [README](https://github.com/ansible-community/ansible-build-data/#renaming-a-collection)

For simplicity, assume that the next minor Ansible release is X.Y.0, and that collection foo.bar with latest release a.b.c has been renamed to baz.bam with latest release A.B.C.
- [x] 1. baz.bam A.B.C must be compatible to foo.bar a.b.c up to renaming plugins. No options must be renamed without backwards compatible aliases, and no defaults or semantics changed.
- [x] 2. baz.bam A.B.C can be added to Ansible X.Y.0.
- [x] 3. A deprecation warning is added to Ansible X.Y.0's changelog (deprecated_features) that foo.bar has been renamed to baz.bam, that Ansible (X+1).0.0 will start having deprecated redirects from foo.bar to baz.bam, and that foo.bar will be removed from a later major release of Ansible.
- [ ] 4. A new release foo.bar (a+1).0.0 is made which contains no more content, but only deprecated redirects to baz.bam. Ideally it will have a dependency on baz.bam so that users that install foo.bar will have working deprecated redirects.
- [ ] 5. Ansible (X+1).0.0 contains both foo.bar (a+1).0.0, and either a baz.bam A.B'.C' release or a later major release that is still compatible with foo.bar a.b.c as specified in 1.
- [ ] 6. foo.bar will be dropped from Ansible (X+2).0.0 (needs to be announced in its changelog as removed_features).

### PR DESCRIPTION
**Pull request by felixfontein**
~~Unfortunately step 4 from the checklist (#306) cannot be implemented since the original repository has been archived already.~~

~~Because of that I marked this PR as a draft, since the changelog entries now don't really make sense.~~

Closes #306.

### CODE REVIEW

**Review by Andersson007 — APPROVED** [review]

File: 8/changelog.yaml
Comment (gotmax23) [inline 8/changelog.yaml:171]:
```suggestion
        For now, both collections are included in Ansible. The content in ``ibm.spectrum_virtualize``
```

File: 9/changelog.yaml
Comment (gotmax23) [inline 9/changelog.yaml:87]:
```suggestion
        For now, both collections are included in Ansible. The content in ``ibm.spectrum_virtualize``
```

**Review by gotmax23 — APPROVED** [review]
Ack. @Shilpi-J, can you take a look here to make sure this looks correct before we merge it?

**Review by Andersson007 — APPROVED** [review]

File: 9/collection-meta.yaml
Comment (sumitguptaibm) [inline 9/collection-meta.yaml:139]: Please change the name to Sumit Gupta (sumitguptaibm) as maintainer has changed.

**Review by sumitguptaibm — CHANGES_REQUESTED** [review]
Please change maintainer as provided in comments.

File: 8/collection-meta.yaml
Comment (felixfontein) [reply 8/collection-meta.yaml:139]:
```suggestion
      - sumitguptaibm
```

File: 9/collection-meta.yaml
Comment (felixfontein) [reply 9/collection-meta.yaml:139]:
```suggestion
      - sumitguptaibm
```

**Review by gotmax23 — APPROVED** [review]

**Review by Andersson007 — APPROVED** [review]
As there were no concerns raised by @sumitguptaibm except the maintainer name, we could merge the PR then. I pinged the original maintainer yesterday by email and she hasn't responded, so we can proceed.
UPDATE: Samit contacted me via email and Shilpi was in copy. Samit said everything looks great. So please folks proceed

**Comment by felixfontein** [comment]
@Andersson007 @gotmax23 @sumitguptaibm @Shilpi-J thanks everyone!

**Comment by Andersson007** [comment]
@felixfontein thanks for the contribution!

**Pull request merged**
